_posts/0000-01-02-alison-lsl.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
What did the monk say to the pilgrim in search of enlightenment? "Shave yourself!"